server/src/test/java/org/elasticsearch/search/aggregations/AggregationsTests.java

```diff
@@ -118,7 +117,6 @@ public class AggregationsTests extends ESTestCase {
         new InternalGeoCentroidTests(),
         new InternalHistogramTests(),
         new InternalDateHistogramTests(),
-        new InternalAutoDateHistogramTests(),
```

Comment:
See https://github.com/elastic/elasticsearch/pull/90294#discussion_r980276526 for reasoning of removal for now.